package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.0.1",
   "repository": "https://github.com/chanzuckerberg/sci-components.git",
   "license": "MIT",
-  "private": "true",
   "devDependencies": {
     "@babel/core": "^7.23.6",
+    "@babel/plugin-syntax-flow": "^7.23.3",
+    "@babel/plugin-transform-react-jsx": "^7.23.4",
     "@babel/preset-env": "^7.23.6",
     "@babel/preset-react": "^7.23.3",
     "@babel/preset-typescript": "^7.23.3",
     "@chanzuckerberg/axe-storybook-testing": "^7.1.3",
     "@chanzuckerberg/story-utils": "^4.0.0",
+    "@czi-sds/components": "workspace:^",
```

Comment:
New yarn workspace handles the cross referenced monorepo package links.

https://yarnpkg.com/features/workspaces#cross-references